source/includes/_accounts.md

```diff
@@ -120,3 +132,78 @@ A successful GET will return an HTTP 200
 <aside class="warning">
 An unsuccessful GET will return an HTTP 404
 </aside>
+
+## Update a specific account
+
+This endpoint updates an individual account
+
+<aside class="warning">
+Updating an account with new usage data erases all existing usage data
```

Comment:
@BlakeWilliams do you think this is a sufficient warning?

Reply:
Sounds good 👍